packages/amplify-codegen/src/utils/downloadIntrospectionSchema.js

```diff
@@ -21,7 +21,10 @@ async function downloadIntrospectionSchema(context, apiId, downloadLocation, reg
       const introspectionDir = dirname(downloadLocation);
       fs.ensureDirSync(introspectionDir);
       fs.writeFileSync(downloadLocation, schema, 'utf8');
-      return relative(amplify.getEnvInfo().projectPath, downloadLocation);
+      if (!context.withoutInit) {
```

Comment:
Do we need this check here? It looks like we call this helper function only for the project which have been initialized

Reply:
True check here is unnecessary. I will remove my changes to this file since we don't download the schema in this workflow